src/EntityFrameworkMock.Moq/EntityFrameworkMock.Moq.csproj

```diff
@@ -15,9 +15,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Castle.Core" Version="4.2.1" />
```

Comment:
Can you avoid updating Moq and Castle.Core packages (here and other csproj files)? Would be better for compatibility.